pkg/domain/entities/volumes.go

```diff
@@ -114,7 +114,8 @@ type VolumePruneOptions struct {
 }
 
 type VolumeListOptions struct {
-	Filter map[string][]string
+	Filter          map[string][]string
+	UntilCmdRequest bool
```

Comment:
Why do we need an extra field?  I'd expect "until" to be a key in the Filter map.

Reply:
If you are cool then we do not need this field. I will correct it.

Reply:
Done